balancer/pickfirst/pickfirst.go

```diff
@@ -115,7 +115,7 @@ func (b pickfirstBuilder) Name() string {
 }
 
 func (pickfirstBuilder) ParseConfig(js json.RawMessage) (serviceconfig.LoadBalancingConfig, error) {
-	var cfg pfConfig
+	var cfg PfConfig
```

Comment:
We should not need to export the config here. In the test, we cast use the `pickfirst` builder into a config parser and make it return the struct for a particular json. Example:
https://github.com/grpc/grpc-go/blob/a2a2023d2a01e28360be9a8a4817b7e83cfcece5/balancer/weightedtarget/weightedtarget_config_test.go#L52

Exporting the config would allow external users to refer to the symbol and removing it would be a breaking change.